front/app/components/ReactionControl/ReactionButton.tsx

```diff
@@ -404,19 +404,17 @@ const ReactionButton = ({
     const projectName = localize(project.data.attributes.title_multiloc);
     const buttonReactionModeIsActive = buttonReactionMode === userReactionMode;
 
-    const describedById = `tooltip-${ideaId}`;
+    const describedById = disabledReason
+      ? `describedbyid-${ideaId}`
```

Comment:
Since it's not used on a tooltip, let's not call id `tooltip` (to avoid confusion).